train/uft/tuning.py

```diff
@@ -50,7 +50,6 @@ def get_params(trial):
     discriminator_adam_lr = trial.suggest_float('discriminator_adam_lr', 1e-6, 1e-2, log = True)
     denoiser_loss_b1 = trial.suggest_float('denoiser_loss_b1', 0.1, 0.9)
     denoiser_loss_b2 = 1 - denoiser_loss_b1
-    print_loss_interval = trial.suggest_int('print_loss_interval', 50, 500)
     denoiser_adversarial_loss_clip_max = trial.suggest_float('denoiser_adversarial_loss_clip_max', 0.0, 1.0)
     denoiser_adversarial_loss_clip_min = trial.suggest_float('denoiser_adversarial_loss_clip_min', 0.0, 1.0)
```

Comment:
- i think clip_min can be negative.
- we need to ensure that clip_min < clip_max always but with this setting, this is not guarantee always and might confuse the model.

i think for now, we can set these parameters to None (we don't clip adversarial loss) because i'm not yet sure how effective this clipping will be. later on, we can add this.

Reply:
ok